From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: royka1 <roykaandorp@gmail.com>
Date: Mon, 6 Oct 2025 23:39:26 +0200
Subject: Add sata node for Orange-Pi-5's overlay (#8707)

> X-Git-Archeology: > recovered message: > * OPi5: SATA power via vcc3v3_pcie20; set to 3.3V; rename overlay
> X-Git-Archeology: > recovered message: > * Convert spaces to tabs
> X-Git-Archeology: > recovered message: > ---------
> X-Git-Archeology: > recovered message: > Co-authored-by: Igor <igor@armbian.com>
> X-Git-Archeology: - Revision 784d367e8aaffa1e2aed7bd844456921b9e8f97e: https://github.com/armbian/build/commit/784d367e8aaffa1e2aed7bd844456921b9e8f97e
> X-Git-Archeology:   Date: Mon, 06 Oct 2025 23:39:26 +0200
> X-Git-Archeology:   From: royka1 <roykaandorp@gmail.com>
> X-Git-Archeology:   Subject: Add sata overlay for Orange-Pi-5 (#8707)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f0504a92e16dc4c4d0ae346cc5c0beb126a00298: https://github.com/armbian/build/commit/f0504a92e16dc4c4d0ae346cc5c0beb126a00298
> X-Git-Archeology:   Date: Sat, 18 Oct 2025 11:34:52 +0200
> X-Git-Archeology:   From: Jianfeng Liu <liujianfeng1994@gmail.com>
> X-Git-Archeology:   Subject: rockchip64-6.18: update patches
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3588s-orangepi-5.dts | 11 ++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-orangepi-5.dts b/arch/arm64/boot/dts/rockchip/rk3588s-orangepi-5.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-orangepi-5.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-orangepi-5.dts
@@ -14,8 +14,8 @@ vcc3v3_pcie20: regulator-vcc3v3-pcie20 {
 		gpios = <&gpio0 RK_PC5 GPIO_ACTIVE_HIGH>;
 		regulator-name = "vcc3v3_pcie20";
 		regulator-boot-on;
-		regulator-min-microvolt = <1800000>;
-		regulator-max-microvolt = <1800000>;
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
 		startup-delay-us = <50000>;
 		vin-supply = <&vcc5v0_sys>;
 	};
@@ -27,6 +27,13 @@ &pcie2x1l2 {
 	status = "okay";
 };
 
+&sata0 {
+	sata-port@0 {
+		target-supply = <&vcc3v3_pcie20>;
+	};
+};
+
+
 &sfc {
 	status = "okay";
 };
-- 
Armbian

